ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.service.spec.ts

```diff
@@ -45,31 +49,35 @@ describe('AddressInputService', () => {
       expect(countryArr[2]).toBeNull();
       done();
     });
-    ais.addressForm.patchValue({country : 'CA'});
+    ais.addressForm.patchValue({ country: 'CA' });
     // unselect country
-    ais.addressForm.patchValue({country : ''});
+    ais.addressForm.patchValue({ country: '' });
   });
+
   it('change in country updates postalCodeLabel$ observable', done => {
     expect(countryServiceSpy.findCountryInfoByCode).not.toHaveBeenCalled();
     ais.postalCodeLabel$.pipe(bufferCount(3)).subscribe(postalCodeLabels => {
       expect(postalCodeLabels.length).toBe(3);
       // there should be a default label. Not worrying about value
       expect(postalCodeLabels[0]).toBeTruthy();
-      expect(postalCodeLabels[1]).toBe('Postal Code');
-      expect(postalCodeLabels[2]).toBeTruthy();
+      expect(postalCodeLabels[1]).toBeTruthy();
+      expect(postalCodeLabels[2]).toBe('Postal Code');
```

Comment:
That's an interesting one - in the Angular 10 postalCodeLabels' items order has changed.